DSharpPlus.Interactivity/InteractivityExtension.cs

```diff
@@ -257,8 +267,11 @@ public async Task<InteractivityResult<ComponentInteractionCreateEventArgs>> Wait
             if (message.Author != this.Client.CurrentUser)
                 throw new InvalidOperationException("Interaction events are only sent to the application that created them.");
 
-            if (message.Components is null || !message.Components.Any(a => a.Components.Any(c => c is DiscordButtonComponent)))
-                throw new ArgumentException("Message does not contain any buttons.");
+            if (!message.Components.Any())
+                throw new ArgumentException("Message doesn't contain any components.");
+
+            if (!message.Components.SelectMany(c => c.Components).Any(c => c.Type is ComponentType.Button))
+                throw new ArgumentException("Message does not contain any select components.");
```

Comment:
Would expect this to to say  "Message does not contain any button components." to be consistent.

Reply:
Oops, forgot to change that